pubspec.yaml

```diff
@@ -20,3 +20,13 @@ dev_dependencies:
   test: ^0.12.3
   _test_annotations:
     path: ./_test_annotations
+
+dependency_overrides:
+  build:
+    git:
+      url: git@github.com:dart-lang/build.git
```

Comment:
1 - why?
2 - It's not gonna survive the PR - I'll have build published before we merge this.